internal/cli/attack_protection_breached_password_detection.go

```diff
@@ -76,12 +79,13 @@ func breachedPasswordDetectionCmd(cli *cli) *cobra.Command {
 
 func showBreachedPasswordDetectionCmd(cli *cli) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "show",
-		Args:    cobra.NoArgs,
-		Short:   "Show breached password detection settings",
-		Long:    "Show breached password detection settings.",
-		Example: `auth0 protection breached-password-detection show`,
-		RunE:    showBreachedPasswordDetectionCmdRun(cli),
+		Use:   "show",
+		Args:  cobra.NoArgs,
+		Short: "Show breached password detection settings",
+		Long:  "Display the current breached password detection settings.",
+		Example: `  auth0 protection breached-password-detection show
+  auth0 ap bpd show --json`,
```

Comment:
Consider in a future PR adding an (additional) example without the `--json` flag (table view), like in other `show` commands.